typescript/sdk/src/core/StarknetCoreModule.ts

```diff
@@ -101,7 +101,7 @@ export class StarknetCoreModule {
     );
 
     // 2. Default Hook - A basic hook implementation for message processing
```

Comment:
If protocol fee hook is now the default hook it should be 2. in order, for 3. then the required hook (also comment needs to be updated here (2. Default Hook -> 3. Required Hook)